src/functions/utils/taxHandlerForOrders.ts

```diff
@@ -5,24 +5,25 @@ import { IInformationsFromJsonFile } from '../../Interface/IInformationsFromJson
 export function handleOrder(specificItem: string, amount: string, baseIngredients: IBaseIngredients[], food: IFood[], informationsFromJSONFile: IInformationsFromJsonFile) {
     //
 
-    let totalCostBeforeTaxes = 0;
+    let costBeforeTaxes = 0;
     const singleIngredient = baseIngredients.find(ingredient => ingredient.name.trim().toLowerCase() == specificItem.trim().toLowerCase());
     const singleDish = food.find(dish => dish.name.trim().toLowerCase() == specificItem.trim().toLowerCase());
-    const taxes: number = informationsFromJSONFile.transactionTax 
+    const taxes: number = informationsFromJSONFile.transactionTax;
 
     if (singleIngredient == undefined && singleDish == undefined) {
         return `We recived malformed input, there's no such ingredient/dish as: ${specificItem}`;
     } else if (singleIngredient != undefined) {
         if (informationsFromJSONFile.order === 'ingredients' || informationsFromJSONFile.order === 'all') {
-            totalCostBeforeTaxes = Math.ceil(singleIngredient.cost * parseInt(amount));
+            costBeforeTaxes = singleIngredient.cost;
         }
     } else if (singleDish != undefined) {
         if (informationsFromJSONFile.order === 'dishes' || informationsFromJSONFile.order === 'all') {
-            totalCostBeforeTaxes = Math.ceil(singleDish?.price * parseInt(amount));
+            costBeforeTaxes = singleDish?.price;
         }
```

Comment:
You already checked if singleDish is not undefined.
You should use:
`costBeforeTaxes = singleDish!.price;`